### PR TITLE
Get last entry

### PR DESCRIPTION
**Pull request by immunity-anomaly**
implement method to get data from db

### CODE REVIEW

File: vosekast_control/TestSequence.py [hunk 1]
```diff
@@ -124,9 +121,8 @@ def __init__(
                     )
                 if self.emulate:
                     # timeout
-                    time_sequence_t1 = datetime.now()
-                    time_sequence_passed = time_sequence_t1 - time_sequence_t0
-                    delta_time_sequence = time_sequence_passed.total_seconds()
+                    time_sequence_t1 = time.time() * 1000
+                    delta_time_sequence = (time_sequence_t1 - time_sequence_t0) / 1000
```
Comment (karlTGA) [inline vosekast_control/TestSequence.py:125]: you store that time not in database. you only check if a time different is bigger then 30s. 
so the change to ms make no sense here and add unnecessary multiplication and devisions.

File: vosekast_control/Scale.py [hunk 2]
```diff
@@ -251,7 +255,8 @@ def read_value_from_scale(self):
 
     def add_new_value(self, new_value):
 
-        self.timestamp = datetime.now()
+        self.timestamp = time.time() * 1000
```
Comment (karlTGA) [inline vosekast_control/Scale.py:258]: why did you store the timestamp in scale class. Are you reusing that value?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+{'': ['*']}
+
+install_requires = \
+['gmqtt>=0.6.2,<0.7.0', 'pendulum>=2.0.5,<3.0.0', 'pyserial>=3.4,<4.0']
+
+extras_require = \
+{':platform_machine != "armv7l"': ['rpi @ '
+                                   'git+https://github.com/nosix/raspberry-gpio-emulator/@master'],
+ ':platform_machine == "armv7l"': ['RPi.GPIO>=0.7.0,<0.8.0'],
+ ':sys_platform == "linux"': ['uvloop>=0.14.0,<0.15.0']}
```
Comment (karlTGA) [inline setup.py:17]: where belongs that setup.py? Until now we dont need that.

File: vosekast_control/TestSequence.py (hunk 1 above)
Comment (immunity-anomaly) [reply vosekast_control/TestSequence.py:125]: makes sense

File: vosekast_control/Scale.py (hunk 2 above)
Comment (immunity-anomaly) [reply vosekast_control/Scale.py:258]: I am only reusing the values in the deque, so I changed it

File: setup.py (hunk 3 above)
Comment (immunity-anomaly) [reply setup.py:17]: the file was created automatically

File: vosekast_control/Scale.py (hunk 2 above)
Comment (karlTGA) [reply vosekast_control/Scale.py]: i only find three places of that timestamp. in the init function. at this place. and short after that where you add that timestamp to the history. So to store that timestamp in the class makes no sense here.

File: setup.py (hunk 3 above)
Comment (karlTGA) [reply setup.py]: I think you can remove this file.

Comment (immunity-anomaly) [reply setup.py]: done

File: vosekast_control/Scale.py [hunk 4]
```diff
@@ -46,7 +46,7 @@ def __init__(
         self.thread_readscale = Thread()
         self.emulate = emulate
         self.is_running = False
-        self.timestamp = datetime.now()
+        self.timestamp = 0
```
Comment (karlTGA) [inline vosekast_control/Scale.py:49]: if you dont use that timestamp, you can delete that property, or?

Comment (immunity-anomaly) [reply vosekast_control/Scale.py]: done

**Pull request merged**